src/main/antlr/Little.g4

```diff
@@ -4,23 +4,25 @@ options {
     language = Java;
 }
 @header {
-    package antlr.main;
+    
+package antlr.main;
+
 }
 
 everything : .*?;
```

Comment:
I don't think the '?' is necessary here, but it also doesn't change the meaning so it is no big deal.